cmds/alias.c

```diff
@@ -17,11 +17,16 @@
 
 #include <hal/hal.h>
 #include <phfs/phfs.h>
+#include <lib/getopt.h>
+#include <lib/log.h>
+
+
+static addr_t aliasBase = 0;
 
 
 static void cmd_aliasInfo(void)
 {
-	lib_printf("sets alias to file, usage: alias [<name> <offset> <size>]");
+	lib_printf("sets alias to file, usage: alias [<-r> <-b BASE>] [<name> <offset> <size>]");
```

Comment:
I don't know what are usage writing guidelines, but I think it should look like this: `[-b <base> | [-r] <name> <offset> <size>]`

Reply:
Way better, thanks, I always have a problem with these